README.md

```diff
@@ -1,2 +1,21 @@
 # hw01
 Repository for homework 01
+
+#**Sarah Jensen**
```

Comment:
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |      x        |            |
| **Coding strategy**         |           |    x          |            |
| **Presentation**            |           |       x       |            |
| **Achievement, creativity** |           |     x         |            |
| **Ease of access**          |    x       |              |            |

Remarks:
* Everyone above me has mentioned this, but I also made the same mistake with the header in forgetting to add the space after the #!
* For the link, you can include it as a hyperlink with the code: [word to hyperlink to] (the link) 
* The picture of the pizza is great, and is embedded at a good size.